clamav/clamav.lua

```
@@ -1,318 +1,297 @@
-	-- Check if we have downloads
-	if
-		not self.ctx.bw.http_content_type
-		or (
-			not self.ctx.bw.http_content_type:match("boundary")
-			or not self.ctx.bw.http_content_type:match("multipart/form%-data")
-		)
-	then
-		return self:ret(true, "no file upload detected")
-	end
+-- Improved boundary extraction function with better quote handling
+function clamav:extract_boundary(content_type)
```

Luacheck (1.2.0)

[warning] unused argument 'self'

(W212)